packages/components/src/ui/ariakit-tooltip/index.tsx

```diff
@@ -22,12 +25,37 @@ function AriaToolTip( props: ToolTipProps ) {
 		children,
 		delay = TOOLTIP_DELAY,
 		placement,
+		position,
 		shortcut,
 		text,
 	} = props;
 
+
+	const DEFAULT_PLACEMENT = 'bottom';
```

Comment:
Interestingly, [the legacy tooltip has a default `position` of `bottom middle` in code](https://github.com/WordPress/gutenberg/blob/8c97cb45b138c5ed10f14eed66923e14870833c6/packages/components/src/tooltip/index.js#L120), while the README states that [the default is `top center`](https://github.com/WordPress/gutenberg/blob/71308e4d69c32a9785b0353418a2074535190b29/packages/components/src/tooltip/README.md?plain=1#L31).

Personally, I think that it makes more sense for a tooltip's default position to be on top of its anchor, but it makes sense to keep `bottom` as the default `placement`, since it corresponds to the current legacy tooltip behaviour